src/content/guides/advanced-cluster-autoscaler-configuration/index.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Cluster Autoscaler advanced configuration"
+description = "Here we describe how you can customize the configuration of the managed Cluster Autoscaler service in your clusters"
+date = "2019-05-09"
+type = "page"
+weight = 40
+tags = ["tutorial"]
++++
+
+# Cluster Autoscaler advanced configuration
+
+The [Cluster Autoscaler addon](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) running inside your cluster has additional configuration options and features that can be customized.
+
+You can customize two of these configuration options on a per cluster basis through a ConfigMap inside your clusters. The ConfigMap is named `cluster-autoscaler-user-values` and is located in the `kube-system` namespace.
+
+__Note:__ This feature is only available in more recent cluster versions. To check if your cluster version supports customization through the ConfigMap, you can check if the above-mentioned ConfigMap is present.
+
+```nohighlight
+$ kubectl -n kube-system get cm cluster-autoscaler-user-values
+NAME                                   DATA      AGE
+cluster-autoscaler-user-values                    0         11m
+```
+
+On cluster creation the ConfigMap is empty and below-mentioned defaults will be applied to the final Cluster Autoscaler deployment. To customize any of the configuration options, you just need to add the respective line(s) in the data field of the user ConfigMap.
+
+__Warning:__ Please do not edit any of the other Cluster Autoscaler related resources. Only the user ConfigMap is safe to edit.
+
+## Scale down utilization threshold
+
+The `scale-down-utilization-threshold` defines the proportion between requested resources and capacity, which under the value Cluster Autoscaler will trigger the scaling down action.
+
+```yaml
+data:
+  scale-down-utilization-threshold: 0.65
+```
+
+Our default value is 65%, which means in order to scale down, one of the nodes has to have less utilization (CPU/memory) than this threshold.
+
+# Expander
+
+When there are several node groups managed by the Cluster Autoscaler we can influence which one will be selected to scale up the cluster when it is needed. Our default is `least-waste`. 
```

Comment:
What is a `node group` here?

Reply:
A node pool. Currently, it is not released yet, but it can handle node groups, and a customer would like to have the configuration ready for when the feature comes

Reply:
If this is about a node pool then I would also call it node pool instead of node group.

But, as node pools aren't there yet, can we remove this and remind ourselves to add the bit of documentation once we are there?

Reply:
ok, I used node group because it is the exact term used in the project. I remove it for now then